tests/MariaDbEventStoreTest.php

```diff
@@ -173,4 +174,28 @@ public function it_ignores_transaction_handling_if_flag_is_enabled(): void
 
         $eventStore->appendTo(new StreamName('Prooph\Model\User'), new ArrayIterator([$streamEvent]));
     }
+
+    /**
+     * @test
+     */
+    public function it_removes_stream_if_stream_table_doesnt_exist(): void
```

Comment:
we should move this test to AbstractPdoEventStoreTest

Reply:
I've written this test case for MariaDB as an example. The query that I used to raise an exception isn't portable between different databases. If you think it should be moved to the parent class, it might be better to write a more common query. Another solution is to write a query with invalid syntax. What do you think?

Reply:
It's also creates a database-specific event store object. I think it's better to write database-specific test cases.